homeassistant/components/heos/quality_scale.yaml

```diff
@@ -0,0 +1,82 @@
+rules:
+  # Bronze
+  action-setup: done
+  appropriate-polling:
+    status: done
+    comment: Integration is a local push integration
+  brands: done
+  common-modules: todo
+  config-flow-test-coverage: done
+  config-flow: done
+  dependency-transparency: done
+  docs-actions: done
+  docs-high-level-description: done
+  docs-installation-instructions: done
+  docs-removal-instructions: todo
+  entity-event-setup: done
+  entity-unique-id: done
+  has-entity-name: done
+  runtime-data: done
+  test-before-configure: todo
+  test-before-setup: done
+  unique-config-entry:
+    status: todo
+    comment: |
+      The HEOS integration only supports a single config entry, but needs to be migrated to use
+      the `single_config_entry` flag. HEOS devices interconnect to each other, so connecting to
+      a single node yields access to all the devices setup with HEOS on your network. The HEOS API
+      documentation does not recommend connecting to multiple nodes which would provide no bennefit.
+  # Silver
+  action-exceptions: todo
```

Comment:
I think it's also good to note that we should raise accordingly with functions in the media player. We now only log, but ideally we raise an exception since the user can view that without going into the logs

Reply:
Comment added to implement!